test/unit-test/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.17)
+
+project(atmospheric_physics VERSION 0.0.1 LANGUAGES Fortran)
+
+find_package(PFUNIT REQUIRED)
+
+set(CMAKE_Fortran_FLAGS "-O0 --coverage")
+if(NOT ATMOSPHERIC_PHYSICS_IS_TOP_LEVEL)
+  message(WARNING "atmospheric-physics is not integrated into the CMake build of any top level "
+                  "project yet and this CMake is for testing purposes only.  "
+                  "Making a change to this project's CMake will not impact the build of "
+                  "a parent project at this time.")
+endif()
+
+option(ATMOSPHERIC_PHYSICS_ENABLE_TESTS "Run pFUnit unit tests" OFF)
+option(ATMOSPHERIC_PHYSICS_ENABLE_CODE_COVERAGE "Run code coverage tool" OFF)
+
+if(ATMOSPHERIC_PHYSICS_ENABLE_CODE_COVERAGE)
+  add_compile_options(-O0 --coverage)
```

Comment:
This is probably my CMake ignorance, but why add these complier options here when it looks like we are already setting them above with the `set(CMAKE_Fortran_FLAGS)` command?

Reply:
Nope, left over from testing.  Removed!